src/cdi_ndi/NDI_AMW.hh

```diff
@@ -0,0 +1,56 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   cdi_ndi/NDI_AMW.hh
+ * \author Ben R. Ryan
+ * \date   2020 Mar 6
+ * \brief  NDI_AMW class definition.
+ * \note   Copyright (C) 2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//----------------------------------------------------------------------------//
+
+#ifndef cdi_ndi_NDI_AMW_hh
+#define cdi_ndi_NDI_AMW_hh
+
+#include "units/PhysicalConstexprs.hh"
+#include <string>
+
+namespace rtt_cdi_ndi {
+
+//============================================================================//
+/*!
+ * \class NDI_AMW
+ *
+ * \brief Class for getting atomic mass weights by ZAID from NDI data using a
+ *        method due to T. Saller. For more details on NDI, see
+ *        https://xweb.lanl.gov/projects/data/nuclear/ndi/ndi.html
+ *        Currently only multigroup data is supported, continuous energy data
+ *        is probably best added through a refactor.
+ */
+//============================================================================//
+class NDI_AMW {
```

Comment:
Is there a reason to use all caps for this class name?  This goes against our normal conventions and makes it easy to mistake this class name with a CPP_MACRO symbol or environment variable.

Reply:
Good point, this was just me casually leaving an acronym as all upper case. I'll switch to `NDI_AtomicMass`.